src/redux/slice/greetingSlice.js

```diff
@@ -3,7 +3,7 @@ import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
 export const fetchGreetings = createAsyncThunk(
     "greeting/fetchGreetings",
     async () => {
-        const response = await fetch("/api/greeting");
+        const response = await fetch("http://127.0.0.1:3000/api/v1/greetings/random");
```

Comment:
- [ ] Good job so far :clap: Unfortunately the react app uses the url and but don't display the random greeting as required for this project because the is url has been blocked by CORS policy . Please fix it in order to make your project work as expected :+1: